src/web/lib/hakuneko/frontend@classic-dark/chapters.html

```diff
@@ -94,12 +94,12 @@
             <table border="0" cellpadding="0" cellspacing="0" width="100%">
                 <tr>
                     <td>
-                        Chapter List 
+                        Chapter List
                     </td>
-                    <td width="1"> 
+                    <td width="1">
                         <i class$="fas fa-fw button [[ getSortClass(chapterSort) ]]" on-click="onToggleSortClick" title="Click to toggle chapter sorting"></i>
                     </td>
-                    <td width="1"> 
+                    <td width="1">
```

Comment:
Not sure why its doing this...?
tried reforking the hole project and removing it from my machine.
but it keeps editing these, bug?!

Reply:
Removing the trailing space seems correct to me, this is in compliance with the current lint configuration.
Unfortunately linting is not yet supported for HTML files, so thats the reason the trailing whitespace was not detected as error.
Your IDE may see the lint configuration and adjust the errors automatically according to the configuration, which is correct.